Makefile

```diff
@@ -0,0 +1,16 @@
+FSFLAG = -lstdc++fs
+TARGET = bin2c
+FILE = bin2c.cpp
+
+$(TARGET):
+	$(CXX) -O3 -std=c++14 $(FILE) $(FSFLAG) -o $(TARGET)
```

Comment:
Can you have it check if ``$(CXX)`` == ``clang`` or ``clang++`` and if it is set ``FSFLAG`` to ``-lc++experimental`` oh and do we need to add ``-lstdc++`` as well for the standard library for g++? I know on clang instead of ``-lstdc++`` one must replace that with ``-lc++``. At least that is how MAC seems to be. Also I would like to remove the Visual Studio files as well.

I am also thinking on setting this on ``-std=c++17`` too. And then have everything be accessed with a namespace named ``fs`` and that is defined like so:

```cpp
#if __has_include(<filesystem>)
#include <filesystem>
#else
#include <experimental/filesystem>
#endif
#ifdef __cpp_lib_filesystem
namespace fs = std::filesystem;
#elif defined(__cpp_lib_experimental_filesystem)
namespace fs = std::experimental::filesystem;
#elif !defined(__cpp_lib_filesystem) &&                                        \
    !defined(__cpp_lib_experimental_filesystem) &&                             \
    (__has_include(                                                            \
        <filesystem>) || __has_include(<experimental/filesystem>)) && defined(_WIN32)
// Visual Studio 2017 (15.7.x) seems to
// not have a c++ std::experimental::filesystem macro
// when std::filesystem is not present,
// and an std::filesystem macro when
// std::experimental::filesystem is
// not present. Because of this I guess
// fallback to std::experimental::filesystem
// and hope it works (/tableflip).
// However it seems the current preview (15.8.x) does.
namespace fs = std::experimental::filesystem;
#endif
```

Reply:
@MateuszOkulus Mind reenabling all the CI's in ``.doozer.json`` file that are commented out and update them and the CI yml files to call use the makefile? That way I know that the build passes by using ``make`` on OSX and Windows (which they should) like it does on Linux.

Reply:
nvm I will do that.